### ISSUE TITLE
Hebrew addon - Left-to-Right Brackets issue

### ISSUE THREAD

**Issue by mac3000** [AnySoftKeyboard/AnySoftKeyboard#588]
### Steps to reproduce
1. Install Hebrew extension
2. Write something in Hebrew, with a word in brackets ()
3. Wrong bracket is added

### Expected behaviour
Correct bracket added - (test) - this is just an example in English, to describe what happens

### Actual behaviour
Tell us what happens instead - )test( - this is just an example in English, to describe what happens

**Android OS version:**
Android 5.1.1

**Device manufacturer and model:**
Samsung Note 5 (SM-N9208)

**List of installed add-ons (like languages, or themes):**
Hebrew
AnySoftKeyboard 3D Theme

------------------------
I am having problems on my new phone with the application.
I recently started using Samsung Note 5, installed the app and also the Hebrew "extension".

For some reason on Hebrew when I use Brackets (), it enters the wrong one.
So, instead of (test) it writes )test( - this is just an example in English, so you know what I'm talking about :)

I tried to change the "Disable Left-to-Right Handling" option, but it doesn't help.

Any ideas?
Let me know what info you need...

**Comment by menny** [comment]
I think it is an issue with some of samsung devices.
Have youvused AnySoftKeyboard with any other device ?

**Comment by mac3000** [comment]
Yes.
I used it on Nexus5.

So it's a known issue?
Hope it gets fixed, since it's annoying.

**Comment by menny** [comment]
You didn't experience that problem with the Nexus, right?

I'll add/fix an option in settings to handle Samsung devices.

**Comment by mac3000** [comment]
Was working fine on Nexus - sorry, forgot to mention that :)

By the way, that issue occasionally happens also when on English for some reason...

Thanks

**Comment by mac3000** [comment]
There is no Activity on this topic since you guys did not update anything.
If the intention is to check if this is still happening, then the answer is not only yes but also that it happens on my new phone - Galaxy Note 10 Plus

So, if you will leave this as a "known issue" for Samsung devices and not fixed (not sure why, since Samsung devices are quite popular), please update with that comment and close this topic. Otherwise I will be waiting for an update.

Thanks

**Comment by menny** [comment]
indeed, it seems to only happen on Samsung phones. Also, not all Samsung phones....

**Comment by AssafTzurEl** [comment]
Veteran user of ASK (thanks, Menny!). New Samsung A53, experiencing wrong parentheses for the first time. FYI.
